packages/launch-editor/get-args.js

```diff
@@ -35,6 +35,7 @@ module.exports = function getArgumentsForPosition (
     case 'code':
     case 'code-insiders':
     case 'Code':
+    case 'vscodium':
```

Comment:
Thanks! Sorry for the late review.
Seems the binary was renamed to `codium` shortly after the PR was opened:
https://github.com/VSCodium/vscodium/pull/176

```suggestion
    case 'codium':
```